### DISCUSSION TITLE
Apakah tabel perlu dilampirkan dalam laporan?

### DISCUSSION THREAD

**Discussion by taruma** [hidrokit/hidrokit#152] [Umum]
### Ringkasan

Diskusi: Apakah tabel perlu dilampirkan dalam laporan?

Hal ini terkait pada laporan utama/teknis/analisis biasanya melampirkan tabel. Akan tetapi kecenderungan penggunaan tabel untuk dapat diverifikasi proses perhitungan oleh pihak ketiga atau evaluasi. Ide saya, apakah memungkinkan untuk memberikan alternatif dari tabel kedalam bentuk yang lebih praktis dan mudah dibaca (grafik)? Atau ini terkait kebiasaan atau ada unsur kepentingan tertentu terkait pelampiran tabel?

Tabel yang saya maksud bisa berupa (hanya contoh saja):
- Informasi statistik (proses perhitungan, tiap kolomnya seperti (X-X_mean)
![image](https://user-images.githubusercontent.com/1007910/155247252-010d45a1-e9e2-43c5-8553-0426adb4b395.png)

- IDF
![image](https://user-images.githubusercontent.com/1007910/155247332-07ae09dc-899b-4886-ac23-27c9aff42a08.png)

Kedua contoh diatas, saya rasa bisa diganti dalam bentuk grafik atau tabel hasil perhitungan yang diperlukan.

**Comment by iingLK** [comment]
Mengikuti lazimnya dalam pembuatan laporan, biasanya tabel itu ditampilkan. Kalau saya, apabila jumlah baris nya sangat panjang saya bisa truncate menjadi hanya, misal 5 baris, namun full tabel nya dijadikan lampiran.

Tidak semua tabel bisa dijadikan grafik, dalam artian, ada tabel yang tidak berguna apabila dijadikan grafik. Karena biasanya grafik menjadi lebih bermakna apabila kita tambahkan grafik tambhannya, i.e. garis hreshold, garis pembanding dll. Namun apabila memang bisa dibuat grafik dan lebih bisa memudahkan pembaca untuk mengetahui dan mengemulasikan kenyataan secara fisik di dalam pemikirannya, menurut saya hal hal tersebut (menampilkan grafik) sah-sah saja.

Atau bisa saja, selama ini tidak ada yang membuat grafik karena memang belum pernah ada yang mencobanya, atau memang kalaupun ada grafiknya, si pembaca masih cukup kesulitan untuk grasp apa yang mau disajikan. Jadi terkadang diberi dalam bentuk tabel... toh kalau dalam bentuk angka biasanya orang tidak bisa membayangkan apa-apa. 

Kalau dikasih grafik, saat masuk ke kepala manusia, akan muncul pertanyaan "apa ini?" lalu dilanjutkan dengan pertanyaan. Namun jika kita menyajikan dalam bentuk tabel, pertanyaan "apa ini?" hanya ada dalam bentuk pemikiran dan tidak pernah dikeluarkan. Tapi setiap manusia bisa meresponnya dengan cara yang berbeda.
